app/models/website.rb

```diff
@@ -66,7 +82,7 @@ def valid_origin?(value)
     return false if origins.blank? # no origins to check against
     return false if value.blank?
     value = value.downcase
-    origins.include?(value)
+    return !origins.include?(value)
```

Comment:
there is no need for return here. Use the rubocop styleguide please!. You add return only in the lines above but this is the last. The last line never needs a return it's understandable.